content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -77,32 +82,45 @@ The Go tracer includes support for the following data stores and libraries.
+| [Redis (go-redis) v8][131]| Fully Supported | [github.com/DataDog/dd-trace-go/contrib/go-redis/redis.v8/v2][93]                |
+| [Redis (go-redis) v9][133]| Fully Supported | [github.com/DataDog/dd-trace-go/contrib/redis/go-redis.v9/v2][134]               |
 | [Redis (redigo)][36]    | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/garyburd/redigo/v2][94]                   |
 | [Redis (new redigo)][38]| Fully Supported | [github.com/DataDog/dd-trace-go/contrib/gomodule/redigo/v2][95]                   |
 | [SQL][40]               | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/database/sql/v2][96]                      |
 | [SQLx][42]              | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/jmoiron/sqlx/v2][97]                      |
+| [PostgreSQL (pgx v5)][135] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/jackc/pgx.v5/v2][136]                  |
 | [MongoDB][44]           | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/go.mongodb.org/mongo-driver/mongo/v2][98] |
 | [MongoDB (mgo)][114]      | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/globalsign/mgo/v2][99]                    |
+| [Gorm][137]             | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/jinzhu/gorm/v2][138]                      |
+| [Gorm v1][69]           | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/gorm.io/gorm.v1/v2][110]                   |
 | [BuntDB][47]            | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/tidwall/buntdb/v2][100]                    |
 | [LevelDB][49]           | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/syndtr/goleveldb/leveldb/v2][101]          |
 | [miekg/dns][51]         | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/miekg/dns/v2][102]                         |
-| [Kafka (confluent)][53] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/confluentinc/confluent-kafka-go/v2][103]   |
+| [Kafka (confluent)][53] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/confluentinc/confluent-kafka-go/kafka/v2][103]   |
+| [Kafka (confluent v2)][53] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/confluentinc/confluent-kafka-go/kafka.v2/v2][139] |
 | [Kafka (sarama)][55]    | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/IBM/sarama/v2][104]                     |
+| [Kafka (sarama v1)][140] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/IBM/sarama.v1/v2][141]                  |
+| [Kafka (Shopify sarama)][142] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/Shopify/sarama/v2][143]             |
+| [Kafka (segmentio)][144] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/segmentio/kafka-go/v2][145]             |
+| [Kafka (segmentio v0)][144] | Fully Supported | [github.com/DataDog/dd-trace-go/contrib/segmentio/kafka.go.v0/v2][146]         |
```

Comment:
I don't think there should be 2 of these (not sure which one is the correct one though)